### ISSUE TITLE
Recognizing as gVCF a standard VCF file

### ISSUE THREAD

**Issue by inti4digbi** [PharmGKB/PharmCAT#174]
* **Do you want to request a *feature* or report a *bug*?**

Bug

* **What is the current behavior?**

``` 
$ docker run --rm -v /Users/intipedroso/tmp_data/:/pharmcat/data pgkb/pharmcat python3 pharmcat_vcf_preprocessor.py -vcf  data/sample.vcf
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
PharmCAT VCF Preprocessor version: 2.9.0
data/sample.vcf is a gVCF file, which is not currently supported.
The PharmCAT VCF Preprocessor will support block gVCF in the future.
```

* **If the current behavior is a bug, please provide the steps to reproduce and, if possible, your example input data** via a [Gist](https://gist.github.com) or similar.

When running the preprocessor or the pipeline I get the warning that input file is a gVCF. The input file is a standard VCF files containing array/chip genotypes + imputed markers. 

* **What is the expected behavior?**
I guess it would be to process the file as a standard VCF file

* **What is the motivation / use case for changing the behavior?**

* **Please tell us about your environment:**
  
  - PharmCAT Version: 2.9
  - JDK Version: [ JDK8 | OpenJDK | etc ]
  - Environment: docker as per PharmCat documentation

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, e.g. stackoverflow, gitter, etc.)

**Comment by inti4digbi** [comment]
Hi, yes i can provide the first part of the file but I am not sure it is needed on this case. 

I have checked and as you said, the variants with a `END=123456` in the `INFO` field trigger the error. This field as part of the standard vcf format description as far as I know, as per https://samtools.github.io/hts-specs/VCFv4.2.pdf section 1.4.1 number 8. 

Is there a preferred practice to deal with this?

**Comment by inti4digbi** [comment]
Hi, here is an example

```
1   944010  rs764300897 GGA G   .   .   END=944012
```
The `END` field is mostly used to track the end position on the original annotation file. We can ignore it for this analysis.

**Comment by BinglanLi** [comment]
The fix for #199 addresses the issue here as well. The preprocessor now recognizes the `END` field used for variant annotations and will not confuse it with a gVCF reference block.
